### PR TITLE
Update Kesmai.mapproj

### PR DESCRIPTION
**Pull request by Tenscore**
addressing Altar exit issue.
adding torchlight on -3.5.

### CODE REVIEW

**Review by Chilly021 — DISMISSED** [review]

**Review by Leolus — APPROVED** [review]

File: Content/Data/Body-External.xml [hunk 1]
```diff
@@ -852,4 +852,34 @@
...
+    <source>(0,0,60,60)</source>
+  </body>
+  <!--Bugbear w kopesh -->
+  <body id="468" corpse="5468">
+    <texture>Tenser\monsters\bugbear\bugbear3</texture>
+    <source>(0,0,80,80)</source>
+  </body>
+  <!--Bugbear w glaive -->
+  <body id="469" corpse="5468">
+    <texture>Tenser\monsters\bugbear\bugbear3</texture>
```
Comment (Chilly021) [inline Content/Data/Body-External.xml:882]: I think you meant to use bugbear4 here.

**Review by Chilly021 — COMMENTED** [review]
I think you wanted to have bugbear 4 on line 873?

**Review by Chilly021 — COMMENTED** [review]
Sorry 882. You will notice two bugbear3s. Looking at the comment and then the art, the bugbear4 art has the khopesh.

File: Content/Data/Body-External.xml (hunk 1 above)
Comment (Tenscore) [reply Content/Data/Body-External.xml]: got it.

**Review by Chilly021 — APPROVED** [review]

**Review by Leolus — APPROVED** [review]

**Review by sproatba — APPROVED** [review]

**Pull request merged**